sympy/solvers/pde.py

```diff
@@ -898,7 +898,7 @@ def pde_separate_add(eq, fun, sep):
     Helper function for searching additive separable solutions.
 
     Consider an equation of two independent variables x, y and a dependent
-    variable w, we look for the product of two functions depending on different
+    variable w, we look for the sum of two functions depending on different
     arguments:
 
     `w(x, y, z) = X(x) + y(y, z)`
```

Comment:
Thanks for taking a look at this.  Actually I'm not familiar with the `pde` module at all.  I only noticed this glitch in the documentation on my first cursory scan of the documentation and assumed that it was a simple copy'n'paste error from the sibling helper function `pde_separate_mul`.

I'll take a closer look and see; I expect that if the help-string for this is changed then that for the sibling should be too.

My guess though was that it should say 'three independent variables x, y, and z, and a dependent variable w', rather than introducing F in place of w.  Having the second function Y on the right-hand side in upper case is a correct fix too.  But I'll take a look at the example before going further.